JOSESwift/Sources/DeflateCompressor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  DeflateCompressor.swift
+//  JOSESwift
+//
+//  Created by Florian Häser on 24.12.18.
+//
+
+import Foundation
+
+internal protocol DeflateCompressor {
```

Comment:
Maybe I'm missing something. Does this protocol do anything / is it used somewhere where it brings abstraction?

Reply:
No you are right. It doesn't bring any benefit. Will remove it.